.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2.1
+jobs:
+  build_android:
+    docker:
+      - image: circleci/android:api-28
+    environment:
+      JVM_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - restore_cache:
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum "app/build.gradle" }}
+      - run:
+          name: Download Dependencies
+          command: ./gradlew androidDependencies
+      - save_cache:
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum "app/build.gradle" }}
+          paths:
+            - ~/.gradle
+      - run:
+          command: cp app/key.properties.sample app/key.properties
+      - run:
+          name: Generate signing key
+          command: keytool -genkey -v -keystore app/debug.keystore -alias AndroidDebugKey -keyalg RSA -keysize 2048 -validity 1 -storepass android -keypass android -dname "cn=TEST, ou=TEST, o=TEST, c=TE"
+      - run:
+          name: Build APK
+          command: ./gradlew clean build
```

Comment:
Is there a need for the "clean" command here (i.e. is there any leftover from previous builds)?

Reply:
Doesn't hurt.